Comanche055/EXTENDED_VERBS.agc

```diff
@@ -632,60 +621,59 @@ STROKON		CS	FLAGWRD6	# V68	PERMITTED ONLY DURING TVC
 		BZMF	ALM/END		# NOT TVC....FLASH OP ERROR LIGHT
 		CAF	PRIO30		# JOB REQUEST, TO SET UP STROKE TEST,
 		TC	NOVAC		#	INCLUDING INITIALIZATIONS
-		SBANK=	PINSUPER	# Added RSB 2009.
```

Comment:
@oldmud0 @wopian Again here in this file, I removed all occurrences with `# Added RSB 2009` remark. 🙏